### ISSUE TITLE
Don't handle '<' inside <pre> as a tag

### ISSUE THREAD

**Issue by tsipinakis** [akrennmair/newsbeuter#353]
If a `<`(or, to be more accurate `&lt;`) appears inside a `<pre>` tag the renderer will parse it as tag start and will ignore the rest of the tag contents.

Example:
With the this HTML:
```
Start
&lt;pre&gt;
for(int i = 0; i &lt;= 10; i++)
&lt;/pre&gt;
End
```
The following is rendered:
```
Start

 for(int i = 0; i
 End
```

**Comment by Minoru** [comment]
I guess you meant this HTML:
```
Start
<pre>
for(int i = 0; i <= 10; i++)
</pre>
End
```
Otherwise I don't understand how you got that output you show.

Anyway, this is invalid markup, so Newsbeuter is free to render it the best it can, which isn't the best it could :) Guess we'll have to add another kludge to the renderer.

**Comment by tsipinakis** [comment]
I used `&lt;` to get that. You can find the full test data [here](https://gist.githubusercontent.com/tsipinakis/9cdfe8c0f8bfabd0ae767106054eb476/raw/e66dfa18857f416f0808b98a313dc987ace02c8e/atom-test-data)
If I try and render with
```
Start
<pre>
for(int i = 0; i <= 10; i++)
</pre>
End
```
It renders as:
```
Start

 for(int i = 0; i = 10; i++)
 /pre
 End
```

**Comment by Minoru** [comment]
Ah, I see. (You shouldn't have removed the link to Gist—example code is always welcome!)

**Comment by tsipinakis** [comment]
Yeah I edited the previous comment because I misunderstood your reply. [Here are my test data for reference](https://gist.github.com/tsipinakis/9cdfe8c0f8bfabd0ae767106054eb476)
